### ISSUE TITLE
Fatal Error after Update from v3.12.0 to v4.0.0

### ISSUE THREAD

**Issue by cakebake** [einpraegsam/powermail#174]
Steps to Reproduce

- Updated Extension over the backend
- Compare current database with specification
- Opened page with a simple contact form in the frontend

> TYPO3 v7.6.22

```
PHP Catchable Fatal Error: Argument 3 passed to In2code\Powermail\ViewHelpers\Misc\CreateRowTagsViewHelper::renderStatic() must be an instance of TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface, instance of TYPO3\CMS\Fluid\Core\Rendering\RenderingContext given, called in /html/typo3_dev/typo3conf/ext/powermail/Classes/ViewHelpers/Misc/CreateRowTagsViewHelper.php on line 39 and defined in /html/typo3_dev/typo3conf/ext/powermail/Classes/ViewHelpers/Misc/CreateRowTagsViewHelper.php line 52
```

**Comment by einpraegsam** [comment]
Pls write me your TYPO3 version and if it runs in composer mode or not (I think not if you updated the extension in backend).
4.0.4 should be 4.0.0 I think

**Comment by cakebake** [comment]
The TYPO3 v7.6.22 instance does not run under composer.

**Comment by einpraegsam** [comment]
Thx, that helps me. I will fix it asap.

**Comment by cakebake** [comment]
I thank you in advance!

**Comment by einpraegsam** [comment]
Thx for the info - should be fixed in 4.0.1
